### DISCUSSION TITLE
Impossibile installare Hacs su Home Assistant Core su Raspberry

### DISCUSSION THREAD

**Discussion by IronMel** [indomus/forum#2443] [GENERALE - Discussioni generiche]
Ciao a tutti, ho installato Home Assistant Core su Docker su Rasberry Pi OS. Dopo aver seguito passo passo le guide (e dopo aver fatto due installazioni da zero di tutto il sistema), continuo ad avere problemi con Hacs. 

Nell'interfaccia di Home Assistant non è visibile e non è ricercabile. Alcune guide dicono di attivarlo tramite addon ma non ho neanche la schermata degli add-on pur avendo attivato la modalità avanzata...

Ho controllato anche[ la guida ufficiale](https://hacs.xyz/docs/setup/download/) e provato con l'url alternativo... nulla!

![image](https://user-images.githubusercontent.com/97842570/234360638-f2bbcc98-4871-4d42-83c3-c857010ec0da.png)

Qualcuno riesce ad aiutarmi? Grazie

**Comment by Scialla** [comment]
Domanda: perché tenti di eseguire il download dello script per l'installazione di HACS da dentro il container? Se installi qualcosa dentro il container, al primo aggiornamento del container la perdi. Dove hai visto indicazioni di farlo da dentro al container? E' sbagliato, devi farlo fuori dal container, nella directory mappata per homeassistant

Non hai la schermata degli addon perché, molto probabilmente, hai installato Homeassistant Core, che NON ha gli addon (non sono indispensabili)

**Reply by snakuzzo** [reply]
Non è sbagliato. Può farlo da dentro o da fuori, tanto finisce nella config dir di HA che è persistente anche quando ricrei il container

**Reply by snakuzzo** [reply]
Hai problemi di connettività

**Reply by snakuzzo** [reply]
Più che altro, perché stai contattando l'endpoint con l'ipv6?

**Reply by snakuzzo** [reply]
Disabilita ipv6

**Reply by IronMel** [reply]
> Disabilita ipv6

sto provando ma esce questo messaggio: 

![image](https://user-images.githubusercontent.com/97842570/235294430-2108abca-2a55-40b1-ba33-30df3ab27e2f.png)

**Reply by snakuzzo** [reply]
Guarda che quello non è un comando, ma è il file da modificare per disabilitare ipv6 ... E che va eseguito come root (o con sudo)

**Reply by snakuzzo** [reply]
```
sudo nano /etc/sysctl.conf
```

E aggiungi questo alla fine

```
net.ipv6.conf.all.disable_ipv6=1
net.ipv6.conf.default.disable_ipv6=1
net.ipv6.conf.lo.disable_ipv6=1
```

Salvi e poi dai questo comando

```
sudo sysctl -p
```
